docs/dev_guide/contents/newcomers.rst

```diff
@@ -17,6 +17,15 @@ If you have any questions, comments, or just want to say hello, we have an activ
 How to Contribute to sunpy
 ==========================
 
+.. _newcomers_generative_ai:
+
+Restrictions on Generative AI Usage
```

Comment:
Would this be better as a note or callout box rather than it's own section?

Reply:
Yeah that is better.